expfmt/text_create.go

```diff
@@ -44,7 +45,7 @@ const (
 var (
 	bufPool = sync.Pool{
 		New: func() interface{} {
-			return bytes.NewBuffer(make([]byte, 0, initialBufSize))
+			return bufio.NewWriterSize(ioutil.Discard, initialBufSize)
```

Comment:
Following up to my initial comment: It's probably fine to simply use `bufio.NewWriter` here and get rid of `initialBufSize` altogether. Let's trust the authors of `bufio` to pick a generally sweet spot for the buf size. If we ever feel we need a different size, we can still introduce it.

Reply:
Changed to the default bufio size.